powerauth-restful-server-javaee/src/main/java/io/getlime/security/powerauth/app/rest/api/javaee/provider/DefaultCustomActivationProvider.java

```diff
@@ -49,4 +50,16 @@ public String lookupUserIdForAttributes(Map<String, String> identityAttributes)
     public boolean shouldAutoCommitActivation(Map<String, String> identityAttributes, Map<String, Object> customAttributes, String activationId, String userId) {
         return true;
     }
+
+    @Override
+    public Integer getMaxFailedAttemptCount() {
```

Comment:
We need to pass some context attributes as we do to the methods above. Otherwise, there is no way to decide on different values. If we only allow constant value, there is no point of doing this, since this can be achieved with a server config.

The context should have the following attributes:

- `userId`
- `identityAttributes`
- `customAttributes`